onnxruntime/core/providers/cpu/tensor/gather_elements.cc

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <string>
```

Comment:
Just curious - why was this header inclusion required now ?

Reply:
It was a lint warning to not include it. I forget the exact text but something about including libraries for types you use.